decentraland/explorer-desktop
# Release 2023 01 30

Release 2023 01 30

## Test steps Desktop (only for this release):
- Open https://renderer-artifacts.decentraland.org/desktop/index.html?dcl://DESKTOP-BRANCH=release/release-20230130&DESKTOP-DEVELOPER-MODE
- Check Custom URL and paste: http://play.decentraland.zone/?explorer-branch=release/release-20230130
- Press Start

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
